src/System.Private.Uri/tests/System.Private.Uri.Tests.builds

```diff
@@ -2,10 +2,16 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="System.Private.Uri.Tests.csproj">
+    <Project Include="FunctionalTests\System.Private.Uri.Tests.csproj">
       <OSGroup>Unix</OSGroup>
     </Project>
-    <Project Include="System.Private.Uri.Tests.csproj">
+    <Project Include="FunctionalTests\System.Private.Uri.Tests.csproj">
```

Comment:
Usually we have the CSPROJ file also have the "Functional" or "Unit" name included.  See System.Net.Http test projects for example.


So, "System.Private.Uri.FunctionalTests.csproj" or "System.Private.Uri.UnitTests.csproj".